README.md

```diff
@@ -1,5 +1,7 @@
 # RoutingServiceProvider #
 
+**Note**: This is just a fork of [marcojanssen/silex-routing-service-provider](https://github.com/marcojanssen/silex-routing-service-provider) with some adaptions for named routes. So all credits belong to [Marco Janssen](https://github.com/marcojanssen). And although Marco Janssen recommends to use it with php (not with yml/xml/etc), I recommend to use it with yaml, because I just tested it with a yaml configuration provided by the ConfigServiceProvider (see below).
```

Comment:
The reason way i recommend php is only because that can be opcached. Using yaml means you should cache it yourself which for a microframework as Silex shouldn't really be needed.